script.js

```diff
@@ -0,0 +1,122 @@
+// Create two variables called width and height.
+// Each of those variables should hold some number.
+// Then, create a new variable that holds the area of a rectangle, calculated based on the width and height variables.
+// Print it to the console.
+
+const width = 9;
+const height = 3;
+
+const rectangleArea = width * height;
+
+console.log(rectangleArea);
+
+// Create two variables called firstNumber and secondNumber.
+// Assign each of them with a different number.
+// Write a program that swaps both values.
+
+let firstNumber = 5;
+let secondNumber = 10;
+
+const temporaryNumber = firstNumber
+
+firstNumber = secondNumber;
+secondNumber = temporaryNumber;
+
+console.log(firstNumber);
+console.log(secondNumber);
+
+// Create variables that hold grades from three different classes:
+//     Physics
+//     Math
+//     English
+// Calculate the average of the grades.
+// If the average equals 4 out of 5 or higher, print "Good job!".
+// Otherwise, print "Try harder!"
+
+let gradePhysic = 4;
+let gradeMath = 4;
+let gradeEnglish = 4;
+
+averageGrades = (gradePhysic + gradeMath + gradeEnglish) / 3;
+if (averageGrades >= 4) {
+    console.log('Good job!');
+} else {
+    console.log('Try harder!');
+}
+
+// Fix the code:
+
+const currentAge = 30;
+const retirementAgeInPoland = 67;
+
+yearsLeftToRetirenment = retirementAgeInPoland - currentAge;
```

Comment:
You are not using neither `let` nor `const` to declare `averageGrades`. That's a big mistake, fix that please